corehq/apps/users/views/mobile/users.py

```diff
@@ -756,11 +763,61 @@ def create_mobile_worker(self, in_data):
             phone_number = self.new_mobile_worker_form.cleaned_data['phone_number']
             couch_user.set_default_phone_number(phone_number)
             send_account_confirmation_sms_if_necessary(couch_user)
+
+        data = self.get_plan_and_user_count_by_domain(self.domain)
+        self.check_and_send_limit_email(self.domain, data['plan_limit'], data['user_count'])
         return {
             'success': True,
             'user_id': couch_user.userID,
         }
 
+    @staticmethod
+    def get_plan_and_user_count_by_domain(domain):
+        subscription = Subscription.get_active_subscription_by_domain(domain)
+        plan_version = subscription.plan_version if subscription else DefaultProductPlan.get_default_plan_version()
+        for rate in plan_version.feature_rates.all():
+            if rate.feature.feature_type == 'User':
+                user_rate = rate
+                break
+        return {
+            'plan_limit': user_rate.monthly_limit,
+            'user_count': FeatureUsageCalculator(user_rate, domain).get_usage()
+        }
+
+    @staticmethod
+    def check_and_send_limit_email(domain, plan_limit, user_count, prev_count=None):
+        if plan_limit == -1:
+            return None
+
+        if prev_count is None:
+            prev_count = user_count - 1
+
+        if user_count >= plan_limit > prev_count:
+            at_capacity = True
+        elif plan_limit > user_count >= (0.9 * plan_limit) > prev_count:
+            at_capacity = False
+        else:
+            return None
+
+        users = WebUser.by_domain(domain)
+        recipients = []
+        for user in users:
+            if user.role_label(domain) == 'Billing Admin' or user.role_label(domain) == 'Admin':
+                recipients.append(user.username)
+        if at_capacity:
+            subject = _("User count has reached the Plan limit for {}").format(domain)
+        else:
+            subject = _("User count has reached 90% of the Plan limit for ({})").format(domain)
+        send_HTML_email(
+            subject,
+            recipients,
+            render_to_string('users/email/user_limit_notice.html', context={
+                'at_capacity': at_capacity,
+                'url': ADDITIONAL_USERS_PRICING
+            }),
+        )
+        return None
```

Comment:
Is this return doing anything?

Reply:
Nope but I think it's good practice to make clear a function won't return anything?

Reply:
I don't think this aligns with our other code. I would disagree that this is good practice, as that would mean putting essentially a no-op return statement at the bottom of every function. Seeing `return None` rather than `return` (which I believe is the intent) confused me a bit, as it implies that this function _might_ return something, and that its callers _might_ care about its return value. I'd advocate to modify the earlier `return None`s to simple `return` statements, and leave off the `return None` at the end